README.md

```diff
@@ -23,13 +23,7 @@ Java, either on the system, or better, via some asdf-java
 Plugin:
 
 ```shell
-asdf plugin add djinni https://github.com/cross-language-cpp/asdf-djinni.git
-```
-
-or, once this plugin will be in the [asdf-plugin repo](https://github.com/asdf-vm/asdf-plugins)
-
-```shell
-asdf plugin djinni
+asdf plugin add djinni
```

Comment:
I don't have asdf installed, but after reading the docs it seems like the command should be `asdf plugin add djinni`. Can you confirm that this is correct?

Reply:
yes, that is correct. Thanks!